packages/rest/package.json

```diff
@@ -44,7 +44,8 @@
     "path-to-regexp": "^2.2.0",
     "qs": "^6.5.2",
     "strong-error-handler": "^3.2.0",
-    "validator": "^10.4.0"
+    "validator": "^10.4.0",
+    "wayfarer": "^6.6.4"
```

Comment:
AFAICT, `wayfarer` is not used by our code, can you remove this dependency please?